beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2055,6 +2057,19 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         })
     }
 
+    pub fn verify_blob_column_sidecar_for_gossip(
+        self: &Arc<Self>,
+        blob_column_sidecar: Arc<BlobColumnSidecar<T::EthSpec>>,
+        subnet_id: u64,
+    ) -> Result<GossipVerifiedBlobColumnSidecar<T>, GossipBlobError<T::EthSpec>> {
+        metrics::inc_counter(&metrics::BLOBS_COLUMN_SIDECAR_PROCESSING_REQUESTS);
```

Comment:
All checks in https://github.com/ethereum/consensus-specs/pull/3574/files#diff-bacd55427cc3606e97b0f11dada895de104a8f9c532aca270efae967199bf261R123 can be implemented except for the crypto, aliasing `verify_data_column_sidecar_kzg_proof` to a noop